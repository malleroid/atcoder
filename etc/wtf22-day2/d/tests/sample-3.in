20
16 6 15 19 1 9 6 1 11 7 6 12 3 11 11 18 10 9 15 5
