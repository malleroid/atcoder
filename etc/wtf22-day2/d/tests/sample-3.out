507808441
401798892
110460932
680359166
737048635
442374434
737773176
980506765
473506608
693729211
532774651
621434128
4273369
839437048
585784927
590354055
969740008
825216624
442091194
660636013
